ConfigSpace/conditions.py

```diff
@@ -134,27 +136,28 @@ def __init__(self, *args):
                 raise ValueError("All Conjunctions and Conditions must have "
                                  "the same child.")
 
-    def get_descendant_literal_conditions(self):
-        children = []
+    def get_descendant_literal_conditions(self) -> List[ConditionComponent]:
```

Comment:
first line of this function initializes a list which is filled and returned. that list might only have one item but it will make the return type list